pkg/crc/oc/clusteroperator.go

```diff
@@ -0,0 +1,75 @@
+package oc
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"time"
+)
+
+var ignoreClusterOperators = []string{"monitoring", "machine-config", "marketplace"}
+
+type ClusterOperator struct {
+	Items []struct {
+		Metadata struct {
+			Name string `json:"name"`
+		} `json:"metadata"`
+		Status struct {
+			Conditions []struct {
+				LastTransitionTime time.Time `json:"lastTransitionTime"`
+				Reason             string    `json:"reason"`
+				Status             string    `json:"status"`
+				Type               string    `json:"type"`
+			} `json:"conditions"`
+		} `json:"status,omitempty"`
+	} `json:"items"`
+}
+
+func GetClusterOperatorStatus(oc OcConfig) (bool, error) {
```

Comment:
`GetClusterOperatorStatus` sounds like it takes an argument - the cluster operator whose status I want to know. Maybe using plural would be OK: `GetClusterOperatorsStatus`. Or `CheckClusterOperatorsAvailable` - because we're not really checking status, just conjunctive availability.

Reply:
@jsliacan I just used what `oc` have, they use `oc get clusteroperator`